src/CONFIG.js

```diff
@@ -78,4 +78,5 @@ export default {
     DEV_PORT: process.env.PORT || 8080,
     E2E_TESTING: lodashGet(Config, 'E2E_TESTING', 'false') === 'true',
     SEND_CRASH_REPORTS: lodashGet(Config, 'SEND_CRASH_REPORTS', 'false') === 'true',
+    IS_USING_WEB_PROXY: expensifyComWithProxy === '/',
```

Comment:
That's the value of the config variable exactly, but we only use the web proxy if we are on the web platform as well.
https://github.com/Expensify/App/blob/e7aae03687588ef2c1df69d39f7a6cf83939762e/src/CONFIG.js#L25

I think it would be more clear to define it like so:
```javascript
IS_USING_WEB_PROXY: getPlatform() === 'web' && useWebProxy,
```